src/Resources/contao/dca/tl_survey.php

```diff
@@ -37,6 +37,7 @@
                     'icon' => 'bundles/mvocontaosurvey/icons/csv.svg',
                     'route' => 'mvo_survey_export',
                 ],
+                'clearRecords' => [],
```

Comment:
Wouldn't it be much simpler to point to the route directly?

```suggestion
                'clear_records' => [
                    'icon' => 'deleteAll.svg',
                    'route' => 'mvo_survey_clear_records'
                ],
```

 Then we would not need a a listener, a form type, an extra template and CSS adjustments :smiley:.

Reply:
But then you would have to use a GET request to trigger an action which you shouldn't do. ;-) If you don't care you can. :-)

Reply:
Yes, I know and agree in general, but think we should ignore it, because that's how the whole BE now works and it's imo not worth the trouble. :upside_down_face:  

I'll put it on my list for the core, though. Maybe we can introduce a `method` button parameter in upcoming versions. So that you could do something like `'my_action' => ['route' => 'foo', 'method' => 'DELETE']`.

Reply:
Okay I reworked it with 
722253d so I hope all is fine now.